### PR TITLE
feat: add enums and interfaces for Apartment

### PR DESCRIPTION
**Pull request by ankit22july**

### CODE REVIEW

File: src/core/propertyTypes/types.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    FOUR_PLUS= "4+",
+};
+
+export enum FURNISH_TYPES {
+    UNFURNISHED= "Unfurnished",
+    Semi_FURNISHED= "Semi-furnished",
+    FULLY_FURNISHED= "Fully-Furnished",
+};
+
+export enum PARKING_TYPES {
```
Comment (koolzsam) [inline src/core/propertyTypes/types.ts:36]: Let's remove _TYPES from these

File: src/core/propertyTypes/types.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    NOT_AVAILABLE= "NA",
+    TWO_WHEELER= "2 Wheeler",
+    FOUR_WHEELER= "4 Wheeler",
+    TWO_PLUS_FOUR_WHEELER= "2+4 Wheeler",
+    INDEPENDENT= "Independent",
+    NON_INDEPENDENT= "Non Independent",
+};
+
+export enum KITCHEN_TYPES {
```
Comment (koolzsam) [inline src/core/propertyTypes/types.ts:45]: Same here

File: src/core/propertyTypes/index.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    fromDate: string;
+    isImmediatelyAvailable: boolean;
+    lastRentAgreementDate: string;
+}
+
+export interface Apartment {
+    address: ApartmentAddress;
+    type: BEDROOM_CATEGORIES;
+    bathroomCount: BATHROOM_COUNT;
+    furnishType: FURNISH_TYPES;
```
Comment (koolzsam) [inline src/core/propertyTypes/index.ts:45]: Can we make it furnish?

File: src/core/propertyTypes/types.ts (hunk 1 above)
Comment (ankit22july) [reply src/core/propertyTypes/types.ts]: removed

File: src/core/propertyTypes/index.ts (hunk 3 above)
Comment (ankit22july) [reply src/core/propertyTypes/index.ts]: done

**Pull request merged**